crates/aptos/CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ All notable changes to the Aptos CLI will be captured in this file. This project
 ### Removed
 - init command from config subcommand is removed. Please use init from the root command.
   - Example: `aptos config init` -> `aptos init`
+### Fixed
+- Panic issue when running `aptos move test` is fixed
```

Comment:
The fix for the `move test` bug is in this release?

Reply:
Looks like it, waiting for the final signal from Move team

Reply:
This is actually the fix, perhaps we should mention this? https://github.com/aptos-labs/aptos-core/issues/8516